extension/jsf-ftest/src/test/java/org/jboss/arquillian/warp/jsf/ftest/lifecycle/TestFacesLifecycleFailurePropagation.java

```diff
@@ -115,24 +116,26 @@ public static WebArchive createDeployment() {
             .addClass(FailingPhaseListener.class);
     }
 
-    @Test(expected = ServerWarpExecutionException.class)
+    @Test()
```

Comment:
Nitpick: no need for brackets if there are no values.

Reply:
Done. Probably there are more... I can do a search if you think so.

Reply:
👍🏼 I can fix post merge anyway. Resolving.

Reply:
I just committed the fix. All were caused by me converting the "expected" attribute.